app/src/main/java/io/multy/viewmodels/WalletViewModel.java

```diff
@@ -108,7 +107,50 @@ public WalletRealmObject createWallet(String walletName) {
             errorMessage.call();
         }
 
+    }
```

Comment:
ViewModel should NOT include any android package classes (e.g. activity). Moe api calls to fragments/classes but avoid using android packages in viewmodels.